src/components/Map/index.tsx

```diff
@@ -39,22 +20,20 @@ import {
   ZINDEX_MARKER_MOUSE_OUT,
   ROUTE,
   SIDO_NAMES,
+  INIT_DUST_INFO,
 } from '@/utils/constants';
+
+import { makeCityMaker, makeSidoMaker, setMakerToNull } from '@/utils/makers';
 import ControlButton from './ControlButton';
 
 const Map = () => {
   const mapRef = useRef<HTMLDivElement | null>(null);
-  const sidoDustInfoMarkers: kakao.maps.CustomOverlay[] = [];
   const [cityDustInfoMarkers, setCityDustInfoMarkers] = useState<
     kakao.maps.CustomOverlay[]
   >([]);
-  const [city, setCity] = useState('동네 정보를 받아오지 못했어요');
-  const [dustInfo, setDustInfo] = useState({
-    fineDustScale: 0,
-    fineDustGrade: 0,
-    ultraFineDustScale: 0,
-    ultraFineDustGrade: 0,
-  });
+  const [selectedSidoOrCity, setSelectedSidoOrCity] =
+    useState('동네 정보를 받아오지 못했어요');
```

Comment:
```suggestion
  const [selectedLocation, setSelectedLocation] =
    useState('동네 정보를 받아오지 못했어요');
```

은 어떨까요?

사실 ID값을 저장하는 거니까 selectedLocationId도 좋을 것 같아요!